server/api/auth.js

```diff
@@ -0,0 +1,37 @@
+const router = require('express').Router()
+const {
+  models: { User }
+} = require('../db/index')
+
+module.exports = router
+
+router.post('/', (req, res, next) => {
+  const { email, password } = req.body
+  User.findOne({ where: { email, password } })
+    .then(user => {
```

Comment:
For catching the error, does it make more sense to:

if (!user) {
 throw new Error('this is the error message')
} else {
 req.session.userId = user.id
 res.json(user)
})
.catch(next)

Reply:
Yea good idea. That will DRY it out. I’ll change it shortly.